app/services/google/search_service.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Google
+  class SearchService
+    SearchServiceError = GoogleScraperRuby::Errors::SearchServiceError
+    HOST = 'www.google.com'
+    # rubocop:disable Layout/LineLength
+    USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36'
+    # rubocop:enable Layout/LineLength
+
+    def self.search!(keyword)
+      uri = URI::HTTPS.build(host: HOST, path: '/search', query: { q: keyword }.to_query)
+      conn = http_client uri
+      response = conn.get(uri, {}, { 'User-Agent' => USER_AGENT })
+      if !response.success?
+        raise SearchServiceError.new(uri,
+                                     "Response with status code #{response.status}")
+      end
+
+      response.body
+    rescue Faraday::ConnectionFailed, Faraday::ServerError, Faraday::ClientError => e
+      raise SearchServiceError.new uri, e
+    end
+
+    def self.http_client(uri)
```

Comment:
This does not need to be a class method:

```suggestion
    private 
    
    def http_client(uri)
```

Reply:
Fixed in 9a60d37461cdc3d701414195e55e9592dd5e84b3